modules/monitoring-creating-user-defined-alerting-rules.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="creating-user-defined-alerting-rules_{context}"]
+= Creating user-defined alerting rules
+
+As a cluster administrator, you can create user-defined alerting rules based on platform metrics. 
+These alerting rules will fire alerts based on the values of chosen metrics.
+
+[NOTE]
+====
+If you create a new, customized `AlertingRule` resource based on an existing platform alerting rule, you should silence the original alert to avoid receiving conflicting alerts.
+====
+
+.Prerequisites
+
+* You are logged in as a user that has the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have enabled Technology Preview features by running `oc edit featuregate/cluster`, and all nodes in the cluster are ready.
```

Comment:
I think
`You have enabled Technology Preview features, and all nodes in the cluster are ready`
is fine enough, same for others

Reply:
OK, Thanks. Updated per your suggestion